docs/analysis-workflows/analysis-overview.md

```diff
@@ -0,0 +1,21 @@
+---
+id: analysis-overview
+title: Harmonization Overview
+sidebar_label: Overview
+---
+
+Data submitted to the the ARGO Data Platform will undergo unified molecular analysis through a harmonized pipeline. Consistent pipelien analysis ensures that data can be considered functionally equvalent across different data sets, including all ARGO Member programs as well as any other initiatives that have adopted the ARGO harmonization pipelines.

```

Comment:
typos 
`pipelien`
`equvalent`

ARGO Member programs - I think we usually say `ARGO programs`